aws/src/main/java/org/apache/iceberg/aws/glue/GlueCatalog.java

```diff
@@ -104,6 +109,8 @@ public GlueCatalog() {
 
   @Override
   public void initialize(String name, Map<String, String> properties) {
+    this.lakeFormationEnabled = PropertyUtil.propertyAsBoolean(properties, AwsProperties.LAKE_FORMATION_ENABLED, false);
```

Comment:
this can be moved to AwsProperties as a method `boolean awsProperties.glueLakeFormationEnabled()`, then we no longer need the additional property of `lakeFormationEnabled`.